Moving all notebooks to the new tutorial

rraymondhp opened the pull request:
I have completed moving all notebooks to the new folder structure

attp commented:
A few issues:
- Remove this file from the commit: appendix/more_qis/.ipynb_checkpoints/wigner_functions-checkpoint.ipynb
- Remove appendix/advanced_qiskit/compiling_and_running.ipynb - @jaygambetta was going to combine this with reference/tools/working_with_backends.ipynb
- Move qiskit-tutorial/reference/approximate/quantum_chemistry.ipynb to appendix/algo_app/  - let @adcorcol figure out how to combine the two into one tutorial on VQE. 
- Remove images/QISKit-c.gif - the file is already in the repository under the name QISKit.gif
- Change the image reference in reference/tools/getting_started.ipynb to QISKit.gif
- Remove  appendix/more_qis/fourier_transform.ipynb, I'm putting that contents in my tutorial for Shor's Algorithm. 

You'll have to make the corresponding changes to the README file.

rraymondhp commented:
Thanks for pointing out the issues. I have removed the QISKit-c.gif file and changed images in the notebooks that used them.

However, for other notebook, because Jay want to finish the rework within this week, let's keep what we have now. We will remove and do as you suggest once we have @jaygambetta and  @adcorcol and you finish writing the new tutorials. I will post them as issues for reminders.
